spec/features/bulk_updates/form_validation_spec.rb

```diff
@@ -18,6 +32,48 @@
     click_on 'Next'
   end
 
+  context 'when viewing the bulk update form before selecting a Field to Update' do
+    context 'when submitting the form before entering any information' do
+      before do
+        click_on 'Preview'
+      end
+
+      it 'displays the appropriate error messages for Name and Update Field' do
+        expect(page).to have_css('#bulk_update_name-error', text: 'Name is required.')
+        expect(page).to have_css('#update_field-error', text: 'Update Field is required.')
+      end
+    end
+
+    context 'when attempting to enter a duplicate name' do
+      before do
+        fill_in 'bulk_update_name', with: used_bulk_update_name
+        select 'Select a Field to Update', from: 'Field to Update'
```

Comment:
this is probably already selected, right?

Reply:
this is to trigger a `blur` on the Name field to trigger the validation error

Reply:
👍